src/main/resources/schemas/schema-form.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "title": "HTTP Redirect",
+    "description": "Send an HTTP redirect response according to specified rules",
+    "properties": {
+        "rules": {
+            "title": "Redirect rules",
+            "description": "Ordered list of rules to apply to inbound request.",
+            "type": "array",
+            "items": {
+                "type": "object",
+                "title": "Rule",
+                "id": "urn:jsonschema:io:gravitee:policy:requestredirect:configuration:Rule",
```

Comment:
id not required. Moreover it causes problems to update a lib in ui-component 😢 ^^